docker/apache-digest-auth-proxy

```diff
@@ -0,0 +1,89 @@
+# Copyright 2017 Intel Corporation
```

Comment:
What should this header be? Obviously the year should be 2019, but are we still using Intel headers at all? Also, this file was added by a contributor (although much of it copy/pasted from an Intel file).

Reply:
If it is substantially derived/copied from another file that has this copyright, this line should remain (with the 2017 date, if we assume it was accurate on the original file). If the file has been substantially modified then adding an additional Copyright line is acceptable/best practice.